stackslib/src/net/api/gettransactions.rs

```diff
@@ -0,0 +1,201 @@
+// Copyright (C) 2013-2020 Blockstack PBC, a public benefit corporation
```

Comment:
This is a bit of a nit, but I think this file should be `gettransaction.rs`, aka non-plural